AdminOpsGuide/dbrepl.rst

```diff
@@ -919,18 +919,18 @@ Download Replication Examples
 
 repl_procedures.tar.gz contains a set of replication example scripts. Each script contains a combination of YottaDB commands that accomplish a specific task. All examples in the Procedures section use these replication scripts but each example uses a different script sequence and diferent script arguments. Always run all replication examples in a test system from a new directory as they create sub-directories and database files in the current directory. No claim of copyright is made with regard to these examples. These example scripts are for explanatory purposes and are not intended for production use. YOU MUST UNDERSTAND AND APPROPRIATELY ADJUST THE COMMANDS GIVEN IN THESE SCRIPTS BEFORE USING THEM IN A PRODUCTION ENVIRONMENT. Typically, you would set replication between instances on different systems/data centers and create your own set of scripts with appropriate debugging and error handling to manage replication between them.
 
-Go to the `Github page <https://github.com/YottaDB/YottaDBdoc/blob/master/AdminOpsGuide/repl_procedures.tar.gz>`_ to download repl_procedures.tar.gz on a test system. 
+Go to the `Github page <https://github.com/YottaDB/YottaDBdoc/raw/master/AdminOpsGuide/repl_procedures.tar.gz>`_ to download repl_procedures.tar.gz on a test system. 
```

Comment:
This file seems to be in "tar" format, not in "gz" format. Need to do a gzip to ensure it is indeed .tar.gz format.

Reply:
Packaged the file again.

Reply:
Sorry to be nitpicky. There are a few empty lines at the end of some of those files. Can we remove them as they offer no added value. And recreate the .gz file. Am thinking if the .gz file is even needed. We can have the scripts separately in the repo and that way we will have hyperlinks for them and avoid the need to repackage the .gz file every time a script changes.

Reply:
This would be a canidate when we move to git lab for this to be in a sub project. we get a tar/zip ball for "free" and we have version control over the files.

Reply:
Oh nice to know. We can revisit the .gz then.

Reply:
We can do that when the repository moves to Gitlab - I'll just put them in the main repository for now.